dbldatagen/column_generation_spec.py

```diff
@@ -16,7 +16,7 @@
     TimestampType, DataType, DateType
 
 from .column_spec_options import ColumnSpecOptions
-from .datagen_constants import RANDOM_SEED_FIXED, RANDOM_SEED_HASH_FIELD_NAME, RANDOM_SEED_RANDOM
+from .datagen_constants import RANDOM_SEED_FIXED, RANDOM_SEED_HASH_FIELD_NAME, RANDOM_SEED_RANDOM, DEFAULT_SEED_COLUMN
```

Comment:
I would recommend applying [`black`](https://github.com/psf/black) formatting globally to the project, this is hard to read

Reply:
interesting suggestion but more on medium term task list

Reply:
re ISSUE TEMPLATE - fixed in separate checkin